ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/RawToDamlLedgerStateReaderAdapter.scala

```diff
@@ -17,7 +17,7 @@ final class RawToDamlLedgerStateReaderAdapter(
   import RawToDamlLedgerStateReaderAdapter.deserializeDamlStateValue
 
   override def read(
-      keys: Seq[DamlStateKey]
+      keys: Iterable[DamlStateKey]
```

Comment:
This change was needed to use a `Map` for the read set, for use in `EqualityBasedPostExecutionConflictDetector`. `readSet.unzip` now returns a `(Iterable[Key], Iterable[Value])`, and I didn't want to force it to a `Seq`.

Reply:
I didn't include it in the summary because there is no semantic change. Can you give me a guideline on why you think this is important enough to call out?

Reply:
Supposing I am going to trace back this change, I am most likely going to wonder how is this related to the PR's rationale and I'm not going to find any explanation there, so I'll have to dig into the PR conversations or to re-review it to find out.

Reply:
Makes sense. I added an extra comment to this PR to explain the change. Thank you for pointing this out.